src/main/java/bio/terra/grammar/azure/SynapseVisitor.java

```diff
@@ -24,22 +24,43 @@ public String visitTable_expr(SQLParser.Table_exprContext ctx) {
     String tableName = getNameFromContext(ctx.table_name());
     String alias = generateAlias(tableName);
     return """
+      (SELECT * FROM
       OPENROWSET(
         BULK '%s',
         DATA_SOURCE = '%s',
-        FORMAT = 'parquet') AS %s
+        FORMAT = 'parquet') AS %s) AS %s
       """
         .formatted(
             FolderType.METADATA.getPath("parquet/%s/*/*.parquet".formatted(tableName)),
             sourceDatasetDatasource,
+            "inner_" + alias,
```

Comment:
I'm not following, why is this modification of the alias required?

Reply:
As far as I can tell, TSQL requires aliasing for every table. So, the inner alias is the alias for the result of the select statement.  Since we're joining on the result of this select statement, the second alias is for the table associated with the join clause. (More about this in my next message). 

However, I actually think I could have just moved the parenthesis and avoided the need for the second alias.  I'll give that a try.

```
(SELECT * FROM
      OPENROWSET(
        BULK '%s',
        DATA_SOURCE = '%s',
        FORMAT = 'parquet') 
) AS %s
```

Reply:
Longer answer to a **different**, but related question-- Why are we adding the "Select *" to this clause? (I should have added something about this in the PR description!) 

With snapshot by query requests, we get need to be able to support joining between tables.  From testing, it doesn't appear that you can just perform joins between different "OPENROWSET" sources. So, I'm adding the "SELECT *" and aliasing the value in order to perform the join. ([Here's a source](https://learn.microsoft.com/en-us/answers/questions/535303/how-to-use-join(inner-full-left-right)-in-synapse); Looks like an alternative could be to first create a table for each OPENROWSET call. This may be worth checking if it improves performance.)

Before these changes:

`
SELECT alias927641339.datarepo_row_id FROM OPENROWSET(
  BULK 'metadata/parquet/vocabulary/*/*.parquet',
  DATA_SOURCE = 'sourceDatasetDataSourceName1',
 FORMAT = 'parquet') AS alias927641339 WHERE alias927641339.vocabulary_id IN ( '1' )
`


With these changes, plus a join:

`
SELECT alias236785828.datarepo_row_id FROM (
SELECT * FROM
OPENROWSET(
  BULK 'metadata/parquet/variant/*/*.parquet',
  DATA_SOURCE = 'sourceDatasetDataSourceName1',
  FORMAT = 'parquet') AS inner_alias236785828) AS alias236785828
JOIN (SELECT * FROM
OPENROWSET(
  BULK 'metadata/parquet/ancestry_specific_meta_analysis/*/*.parquet',
  DATA_SOURCE = 'sourceDatasetDataSourceName1',
  FORMAT = 'parquet') AS inner_alias1748223664) AS alias1748223664
ON alias1748223664.variant_id = alias236785828.id WHERE alias1748223664.variant_id IN ( '1:104535993:T:C' )
`

Reply:
I tested out removing the "inner_alias" alias as I mentioned in the first comment, but it failed with the following error. I'm going to leave it as is! 

> com.microsoft.sqlserver.jdbc.SQLServerException: A correlation name must be specified for the bulk rowset in the from clause.

<img width="705" alt="image" src="https://github.com/DataBiosphere/jade-data-repo/assets/13254229/9684d9cb-ae96-4dc4-badb-2ae34c7fe79d">

Reply:
This is a fix for snapshot by query.